src/dailies.ts

```diff
@@ -541,3 +542,27 @@ export function gingerbreadPrepNoon(): void {
     }
   }
 }
+
+export function hipsterFishing(): void {
+  if (have($familiar`Mini-Hipster`)) {
+    useFamiliar($familiar`Mini-Hipster`);
+  } else if (have($familiar`Artistic Goth Kid`)) {
+    useFamiliar($familiar`Artistic Goth Kid`);
+  } else return;
+
+  while (findRun(false) && get("_gothKidFights") < 7) {
```

Comment:
```suggestion
  while (findRun(false) && get("_hipsterAdv") < 7) {
```

`_gothKidFights` is to track the pvp fights acquired, you want `_hipsterAdv`